actions/websocket_actions.jsx

```diff
@@ -1223,6 +1227,8 @@ function handleChannelViewedEvent(msg) {
 
 export function handlePluginEnabled(msg) {
     const manifest = msg.data.manifest;
+    dispatch({type: ActionTypes.RECEIVED_WEBAPP_PLUGIN, data: manifest});
```

Comment:
Shouldn't we do something also on plugin disabled?

Reply:
This is already handled as `handlePluginDisabled` calls `removePlugin`, which calls `removeWebappPlugin` which dispatches the action to change the redux store: https://github.com/mattermost/mattermost-webapp/blob/f9b4163345d80fcb6d01a574c41c6c3c227e2ec4/plugins/actions.js#L8-L13

For some reason it wasn't being handled when a plugin is enabled. I think the state of the reducer doesn't really matter to most of the application, so it was fine to be out of sync in some way, but now we need it to be correct.

Reply:
I decided to keep this change in since it's a bug fix